.travis.yml

```diff
@@ -67,3 +74,8 @@ notifications:
 branches:
   except:
     - /^0\..*$/
+
+addons:
```

Comment:
Furthermore, each jobs in `matrix.include` defines `addons`, which will override these, so this is effectively a noop.

If you want to use these addons, you'd need to add this to each of the jobs in `matrix.include`.